Mathlib/CategoryTheory/Skeletal.lean

```diff
@@ -92,9 +95,32 @@ instance : (fromSkeleton C).EssSurj where mem_essImage X := ⟨Quotient.mk' X, Q
 -- Porting note: named this instance
 noncomputable instance fromSkeleton.isEquivalence : (fromSkeleton C).IsEquivalence where
 
+variable {C}
+
+/-- The class of an object in the skeleton. -/
+abbrev toSkeleton (X : C) : Skeleton C := ⟦X⟧
+
+/-- The isomorphism between `⟦X⟧.out` and `X`. -/
+noncomputable def preCounitIso (X : C) : (fromSkeleton C).obj (toSkeleton X) ≅ X :=
+  Nonempty.some (Quotient.mk_out X)
```

Comment:
I pushed an explicit construction of the inverse. This could be made to equal `eqToIso` rather than an arbitrary chosen isomorphism when both sides are equal, but that doesn't make the explicit construction easier.